cmd/finch/nerdctl_test.go

```diff
@@ -369,11 +369,10 @@ func TestNerdctlCommand_run(t *testing.T) {
 				getVMStatusC.EXPECT().Output().Return([]byte("Running"), nil)
 				logger.EXPECT().Debugf("Status of virtual machine: %s", "Running")
 				ncsd.EXPECT().LookupEnv("FINCH_DOCKER_COMPAT").Return("", false)
-				logger.EXPECT().Debugf(`Resolving special IP "host-gateway" to %q for host %q`, "192.168.5.2", "name")
 				ncsd.EXPECT().LookupEnv("COSIGN_PASSWORD").Return("", false)
 				c := mocks.NewCommand(ctrl)
 				lcc.EXPECT().Create("shell", limaInstanceName, "sudo", "-E", nerdctlCmdName, "run",
-					"--rm", "--add-host", "name:192.168.5.2", "alpine:latest").Return(c)
+					"--rm", "--add-host", "name:host-gateway", "alpine:latest").Return(c)
```

Comment:
Do we still need these test cases as unit tests if the tested logic is not in the file anymore?

Reply:
will change it to a more meaningful test

Reply:
@ningziwen I already delete those unrelated tests. I add 2 more to the nerdctl_config_applier_test on my latest commit

Reply:
Did you push it? It is not reflected in the PR.

Reply:
Yes, you can check my last commit to the PR

Reply:
@ningziwen can you check again?